src/app/dashboard/filter-ops/filter-ops.component.ts

```diff
@@ -90,11 +90,9 @@ export class FilterOpsComponent implements FilterExecutionProvider {
 
   restoreGroup(group: string, callerIndex: number): void {
     this.getInputGroup(group).forEach((item) => {
-      if (item.index !== callerIndex) {
-        if (!item.valueIsSet()) {
-          if (item.getVal().length > 0) {
-            item.addParam();
-          }
+      if (item.index !== callerIndex && !item.valueIsSet()) {
+        if (item.getVal().length > 0) {
```

Comment:
I think this may still generate a SonarQube warning because you could merge these two if statements.

Reply:
Done